lilio/resampling.py

```diff
@@ -282,18 +325,19 @@ def resample(
     if intervals is None:
         raise ValueError("Generate a calendar map before calling resample")
 
+    _check_valid_resampling_methods(how)
     utils.check_timeseries(input_data)
     # This check is still valid for all calendars with `freq`, but not for CustomCalendar
     # TO DO: add this check when all calendars are rebased on the CustomCalendar
     # utils.check_input_frequency(mapped_calendar, input_data)
```

Comment:
Ah yes I missed that. We cannot bring it back for the shorthands, as they are just fuctions generating a `lilio.Calendar` object.

I think these lines (and the accompanying check) should be removed, and issue #10 can be used for a discussion on how to move forward. For example compare 
```py
min([interval.length for interval in cal.intervals]) > inferred_freq
```

Reply:
Oh yes. This can be solved by adding a identifier when the calendar is created via the shorthands. But I think this sounds a bit clumsy... I think what we need here is simply a reminder to the user when they perform some downscaling operations.

I agree that we can remove these lines and the relevant function in the check and continue our discussion in issue #10.